packages/design-system-scripts/gulp/docs/extractReactData/parseReactFile.js

```diff
@@ -113,8 +113,16 @@ function parseComponent(file, options) {
     if (!options.typescript) {
       docs = [{}];
     } else {
+      // Filter so that when Components extend standard HTML, those props do not also get printed in the docs page
```

Comment:
What this is doing is saying we only want props that are the components, so that we don't print all HTML props when we extend HTML components.